inlong-agent/agent-core/pom.xml

```diff
@@ -84,5 +84,17 @@
             <groupId>io.prometheus</groupId>
             <artifactId>simpleclient_httpserver</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.typesafe.akka</groupId>
+            <artifactId>akka-stream_2.11</artifactId>
+            <version>2.5.21</version>
```

Comment:
In the sub-module, just use the `${xxx.version}` instead of a specific version number.

Reply:
done